ovos_config/mycroft.conf

```diff
@@ -630,36 +664,34 @@
   },
   // turn utterances into phoneme sequences, used to generate mouth movements
   "g2p": {
-     "module": "ovos-g2p-plugin-dummy"
+     "module": "ovos-g2p-plugin-heuristic-arpa"
   },
 
   // DEPRECATED: this section is in the process of being fully removed,
   // only providing default values for clean migration
   "padatious": {
-    "intent_cache": "~/.local/share/mycroft/intent_cache",
-    "train_delay": 4,
-    "single_thread": false,
```

Comment:
Missed in review this is a breaking change https://github.com/OpenVoiceOS/ovos-core/blob/d3abd8b8f18f82b3d6d4264e59269cedbb1e80a8/mycroft/skills/intent_services/padatious_service.py#L140